app/views/shared/_games_card.html.erb

```diff
@@ -10,7 +10,7 @@
       <% if !game.cloud_photo.key.nil? %>
         <%= cl_image_tag game.cloud_photo.key, class: "card-img-top " %>
       <% else %>
-        <%= image_tag game.photo, class: "card-img-top " %>
+        <%= image_tag "placeholder.png", class: "card-img-top " %>
```

Comment:
wasn't sure why game.photo was there; it didn't work for me but we might want to put back in?